lib/screens/overview_screen/widgets/my_health_section.dart

```diff
@@ -41,22 +47,38 @@ class MyHealthSection extends StatelessWidget {
           runSpacing: 20,
           children: <Widget>[
             Container(
```

Comment:
Why won't you introduce an extra widget for the tiles as a refactoring measure?

Reply:
Done!